var/spack/repos/builtin/packages/py-jaxlib/package.py

```diff
@@ -47,6 +47,13 @@ class PyJaxlib(PythonPackage, CudaPackage):
     depends_on("py-absl-py", when="@:0.3", type=("build", "run"))
     depends_on("py-flatbuffers@1.12:2", when="@0.1", type=("build", "run"))
 
+    conflicts("~cuda")
+    conflicts(
+        "cuda_arch=none",
```

Comment:
I think we just want:
```suggestion
        "+cuda cuda_arch=none",
```
and remove the other conflict.